lib/exqlite/basic_api.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Exqlite.BasicAPI do
```

Comment:
Is this for just testing purposes?

Reply:
Currently yes, to keep the API consistent. I feel the current API with Exqlite.Sqlite and Exqlite.Connection is somewhat confusing and there is some overlap, but none of them allow simple and straightforward usage. I could move this module to `test/support` for now, but it might be beneficial to consider providing a simple API for basic usage in the future.

Reply:
The `DBConnection` interface definitely confusing and unfriendly for sure. I think it's okay to introduce, though not a fan of `BasicAPI` as the name just because it's sort of redundant.

I don't know if we want to use something like `SimpleConnection` as a different name. Would that cause confusion with the `Exqlite.Connection` interface?

@kevinlang what's your opinion?

Reply:
At worst, we move this to the `test/support` and once we come to a consensus we could `git mv` it out.

Reply:
I would be fine renaming it. I just did not want to waste my mental cycles on naming yesterday, because you know... naming and cache invalidation ... hehe. We could also move those functions to the top-level module `Exqlite`, that is currently completely empty. But then again, this would encourage this API as the official one, which we might not want just yet.  

Alternative names (in no particular order and mostly )
- Exqlite.Runner
- Exqlite.API
- Exqlite.Basic
- Exqlite.Simple
- Exqlite.Easy
- Exqlite.Base
- Exqlite.Smart

Reply:
I think `Exqlite.Basic` would suffice.

Reply:
Cool.